interfaces/portal/src/app/domains/transaction/transaction.model.ts

```diff
@@ -0,0 +1,27 @@
+// import { TransactionEventInterface } from '@121-service/src/payments/transactions/transaction-events/dto/transaction-event-data.dto';
+import { TransactionEventsReturnDto } from '@121-service/src/payments/transactions/transaction-events/dto/transaction-events-return.dto';
+import { TransactionEventDescription } from '@121-service/src/payments/transactions/transaction-events/enum/transaction-event-description.enum';
+import { TransactionEventType } from '@121-service/src/payments/transactions/transaction-events/enum/transaction-event-type.enum';
+
+import { Dto } from '~/utils/dto-type';
+
+// TODO: this interface is repeated here instead of importing from 121-service, as that gives lint errors down the line, even though the interfaces are exactly equal
```

Comment:
Is there already some generic "refactor these specific patterns"-item on the backlog for these kind of issues?

If we mentions the `AB#000` in all these comments, its easy to quantify later to "fix X issues"...